app/controllers/sessions_controller.rb

```diff
@@ -1,9 +1,17 @@
 class SessionsController < ApplicationController
   def index
+    @sessions = sessions.includes(:attendees, :speakers, :location, :tags)
+    @sessions = @sessions.starts_at(params[:starts_at].to_date) if params[:starts_at].present?
   end
 
   def show
-    @session = Session.find(params[:id])
+    @session = sessions.find(params[:id])
     @speaker = @session.speakers.first
   end
+
+  private
+
+  def sessions
+    @sessions ||= current_conference&.sessions
```

Comment:
No need to memoize queries that aren't expensive, plus you are only calling this method once per action so the memoization isn't being used. 
```suggestion
    current_conference&.sessions
```